portfolio.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <link type="text/css" rel="stylesheet" href="normalize.css">
+    <link type="text/css" rel="stylesheet" href="styles.css">
+</head>
+
+<!--[if lte IE 9]><body class="ieLegacy"><![endif]-->
+<!--[if !IE]><body><![endif]-->
+
+<body>
+    <div>
+        <header class="container">
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                </ul>
+            </nav>
+
+            <h1>My Portfolio</h1>
+        </header>
+
+        <main>
+            <div class="col3 col2Sm">
+                <h2><a href="/sites/academy-php7/html/Responsive site project/index.html">Pilot Shop</a></h2>
```

Comment:
change these to the live links where appropriate and all others to #

Reply:
Changed two to live links ,and the others were changed to #.